sensehat/src/androidTest/java/com/google/android/things/contrib/driver/sensehat/SenseHatDeviceTest.java

```diff
@@ -38,34 +39,38 @@
     @Test
     public void senseHat_DisplayColor() throws IOException {
         // Color the LED matrix.
-        LedMatrix display = SenseHat.openDisplay();
-
-        display.draw(Color.MAGENTA);
-        // Close the display when done.
-        display.close();
+        try (LedMatrix display = SenseHat.openDisplay()) {
```

Comment:
Should we be doing `try` here? If it doesn't work, I think we'd want the test to fail.

Reply:
It will, there is no catch(). I changed this to use the autoclose feature which was implemented but never used / demonstrated by the original author.

Reply:
So what happens if `SenseHat.openDisplay()` fails? Does the test still pass because it's within a `try` block?